CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ execute_process(
     ${Python_EXECUTABLE}
     dump_symbols.py
     "${occt_libs}"
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
```

Comment:
I am trying to package cadquery for guix.

I had the same fix / workaround, until I tried to only change line 36, to the following
`    ${CMAKE_SOURCE_DIR}/dump_symbols.py`
which also "fixed" it for me. I don't know which one is the better "fix"

But I'm here to add my :+1: